mcs/tools/mkbundle/mkbundle.cs

```diff
@@ -872,6 +872,7 @@ static void GenerateBundles (List<string> files)
 			Stream template_stream;
 			if (String.IsNullOrEmpty (mono_api_struct_file)) {
 				tc.WriteLine ("#define USE_DEFAULT_MONO_API_STRUCT");
+				tc.WriteLine ("void mono_aot_register_module (gpointer *aot_info);");
```

Comment:
Adding the line manually to the intervening .c file(using -c with mkbundle), it doesn't work because it can't find gpointer:
```
gcc -o ipy ipy.c `pkg-config --cflags --libs mono-2`
ipy.c:6:32: error: unknown type name ‘gpointer’
 void mono_aot_register_module (gpointer *aot_info);
                                ^
ipy.c: In function ‘init_default_mono_api_struct’:
ipy.c:117:33: warning: assignment from incompatible pointer type [-Wincompatible-pointer-types]
  mono_api.mono_jit_set_aot_mode = mono_jit_set_aot_mode;
                                 ^
ipy.c:118:38: error: ‘mono_aot_register_module’ undeclared (first use in this function)
  mono_api.mono_aot_register_module = mono_aot_register_module;
                                      ^
ipy.c:118:38: note: each undeclared identifier is reported only once for each function it appears in
```